### PR TITLE
Prosthetic limb quirks

### PR DESCRIPTION
**Pull request by BootlegBow**
## About The Pull Request

#33 but not broken.

## Why It's Good For The Game

Same as #33 

## Changelog
:cl:
add: Four new prosthetic limb quirks, allowing players to choose which limb they want to be prosthetic
/:cl:

### CODE REVIEW

File: ModularTegustation/tegu_quirks.dm [hunk 1]
```diff
@@ -0,0 +1,80 @@
+//Prosthetic Limb
+//Right Arm
+/datum/quirk/prosthetic_limb/r_arm
+	name = "Prosthetic Right Arm"
+	desc = "An accident caused you to your right arm. Because of this, you now have a prosthetic arm!"
+	value = -4
+	medical_record_text = "During physical examination, patient was found to have a prosthetic right arm."
+	hardcore_value = 3
+
+/datum/quirk/prosthetic_limb/r_arm/on_spawn()
```
Comment (EgorDinamit) [inline ModularTegustation/tegu_quirks.dm:10]: This proc should be on /datum/quirk/prostethic_limb, so it triggers for every kind of limb and saves space.
Then you just add every possible limb in switch(limb_slot).

**Comment by BootlegBow** [comment]
@EgorDinamit is that good, or have I misunderstood?

**Comment by BootlegBow** [comment]
Looks like I misunderstood.

File: ModularTegustation/tegu_quirks.dm (hunk 1 above)
Comment (BootlegBow) [reply ModularTegustation/tegu_quirks.dm:10]: Wouldn't that randomly select a limb with all 4 quirks?

**Comment by EgorDinamit** [comment]
I'll clean it up.

**Comment by BootlegBow** [comment]
Thanks

File: code/datums/quirks/negative.dm [hunk 2]
```diff
@@ -413,9 +413,11 @@
 	var/slot_string = "limb"
 	medical_record_text = "During physical examination, patient was found to have a prosthetic limb."
 	hardcore_value = 3
+	var/limb_slot // Tegu edit
 
 /datum/quirk/prosthetic_limb/on_spawn()
-	var/limb_slot = pick(BODY_ZONE_L_ARM, BODY_ZONE_R_ARM, BODY_ZONE_L_LEG, BODY_ZONE_R_LEG)
```
Comment (JohnFulpWillard) [inline code/datums/quirks/negative.dm:418]: Dont delete this, just comment it out, that way TG will write over the commented out code if they change it

**Review by JohnFulpWillard — APPROVED** [review]

File: code/controllers/subsystem/processing/quirks.dm [hunk 3]
```diff
@@ -27,7 +27,8 @@ PROCESSING_SUBSYSTEM_DEF(quirks)
 							list("Alcohol Tolerance","Light Drinker"), \
 							list("Clown Fan","Mime Fan"), \
 							list("Bad Touch", "Friendly"), \
-							list("Extrovert", "Introvert"))
+							list("Extrovert", "Introvert"), \
+							list("Prosthetic Limb", "Prosthetic Right Arm", "Prosthetic Left Arm", "Prosthetic Right Leg", "Prosthetic Left Leg")) //Tegu edit
```
Comment (JohnFulpWillard) [inline code/controllers/subsystem/processing/quirks.dm:31]: Dont add this to the bottom of the list, read our readme.md for more

**Comment by JohnFulpWillard** [comment]
I dont like the changelog, since that message will show up in game, please make sure it says exactly what's being added

**Comment by BootlegBow** [comment]
> I dont like the changelog, since that message will show up in game, please make sure it says exactly what's being added

Sorted

**Pull request merged**